cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/template/etl/common/TrackedTransform.java

```diff
@@ -41,4 +42,11 @@ public void transform(IN input, Emitter<OUT> emitter) throws Exception {
     metrics.count("records.in", 1);
     transform.transform(input, emitter);
   }
+
+  @Override
+  public void destroy() {
+    if (transform instanceof Destroyable) {
+      Destroyables.destroyQuietly((Destroyable) transform);
```

Comment:
no particular reason, other than that the caller of this method ``TransformExecutor.destroy`` calls ``Destroyables.destroyQuietly``. I guess I'll have this one just delegate, and let the ``TransformExecutor`` do it quietly?